backend/sirius-web-spring-collaborative/src/main/java/org/eclipse/sirius/web/spring/collaborative/handlers/DeleteRepresentationEventHandler.java

```diff
@@ -76,15 +72,10 @@ public EventHandlerResponse handle(IEditingContext editingContext, IInput input)
             var optionalRepresentation = this.representationService.getRepresentation(deleteRepresentationInput.getRepresentationId());
 
             if (optionalRepresentation.isPresent()) {
-                this.representationService.delete(deleteRepresentationInput.getRepresentationId());
+                RepresentationDescriptor representationDescriptor = optionalRepresentation.get();
+                this.representationService.delete(representationDescriptor.getId());
 
-                var optionalProject = this.projectService.getProject(editingContext.getProjectId());
-                if (optionalProject.isPresent()) {
-                    Project project = optionalProject.get();
-                    eventHandlerResponse = new EventHandlerResponse(false, Tree.class::isInstance, new DeleteRepresentationSuccessPayload(project));
-                } else {
-                    eventHandlerResponse = new EventHandlerResponse(false, representation -> false, new ErrorPayload(this.messageService.projectNotFound()));
-                }
+                eventHandlerResponse = new EventHandlerResponse(false, Tree.class::isInstance, new DeleteRepresentationSuccessPayload(representationDescriptor.getId()));
```

Comment:
Not related to this change, but technically `Tree.class:isInstance` is not precise enough as a predicate. If/when we have actual tree representations elsewhere than in the explorer, this would trigger their refresh.
We should also match on `tree.getDescriptionId().equals(IRepresentationDescriptionService.EXPLORER_TREE_DESCRIPTION`.

```java
    private boolean isExplorerTree(IRepresentation representation) {
        return representation instanceof Tree && Objects.equals(((Tree) representation).getDescriptionId(), IRepresentationDescriptionService.EXPLORER_TREE_DESCRIPTION);
    }
```

Reply:
Done